Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -1355,6 +1355,18 @@ WASM_SLOW_PATH_DECL(i64_trunc_sat_f64_s)
 extern "C" UGPRPair SYSV_ABI slow_path_wasm_throw_exception(CallFrame* callFrame, JSWebAssemblyInstance* instance, Wasm::ExceptionType exceptionType)
 {
     SlowPathFrameTracer tracer(instance->vm(), callFrame);
+#if ENABLE(WEBASSEMBLY_BBQJIT)
+    void* pc = instance->faultPC();
+    instance->setFaultPC(nullptr);
+    auto* callee = static_cast<Wasm::LLIntCallee*>(callFrame->callee().asNativeCallee());
+    if (callee->category() == NativeCallee::Category::Wasm) {
```

Comment:
I think you can probably just `ASSERT` this.

Reply:
Done.